package.json

```diff
@@ -4,10 +4,11 @@
   "license": "MIT",
   "private": true,
   "dependencies": {
-    "chlu-did-service": "ChluNetwork/chlu-did-service",
+    "chlu-did-service": "ChluNetwork/chlu-did-service#fix-chlu-did",
```

Comment:
Should we move this branch dependency out?

Reply:
@kulpreet yes but we got https://github.com/ChluNetwork/chlu-did-service/pull/1 to merge

Reply:
Just merged that, so this should be unblocked. I can just edit it here and merge, might save us time.

Reply:
no, it needs a `yarn` and `yarn upgrade chlu-did-service` run to fix the `yarn.lock`. I just pushed a new commit